server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -185,6 +184,11 @@
    */
   private String syncDir = "data" + File.separator + "system" + File.separator + "sync";
 
+  /**
+   * Performance tracing directory, stores performance tracing files
+   */
+  private String tracingDir = "data" + File.separator + "tracing";
```

Comment:
here should be "TRACING_FOLDER_NAME"..

Reply:
I think you are right. But the Dir strings in this file are all defined this way....